.gitignore

```diff
@@ -2,5 +2,6 @@ node_modules
 dist
 .husky/_
 .husky/.gitignore
+.idea
```

Comment:
Please don't add things related to your IDE...

I suggest you add this to `~/.gitconfig` so you'll never have to do that again:

```
[core]
	excludesFile = ~/.global_gitignore
```

And of course add `.idea` to a new `~/.global_gitignore` file

Reply:
Will do! Thank you for the tip!

Reply:
Go ahead